EasyPost/Client.cs

```diff
@@ -14,8 +14,11 @@ public class Client
         private readonly RestClient _restClient;
         private readonly ClientConfiguration _configuration;
         private string UserAgent => $"EasyPost/v2 CSharpClient/{_libraryVersion} .NET/{_dotNetVersion}";
-        private int _connectTimeoutMilliseconds = 30000;
-        private int _requestTimeoutMilliseconds = 60000;
+
+        private readonly int _defaultConnectTimeoutMilliseconds = 30000;
+        private int? _connectTimeoutMilliseconds;
```

Comment:
By keeping this as a private variable, it means it can only be accessed through the getter/setter correct? Users couldn't access this directly?